web3/version.py

```diff
@@ -1,8 +1,46 @@
+from web3.method import (
+    Method,
+)
 from web3.module import (
     Module,
+    ModuleV2,
 )
 
 
+class BaseVersion(ModuleV2):
+    retrieve_caller_fn = None
+
+    _get_node_version = Method('web3_clientVersion')
+    _get_protocol_version = Method('eth_protocolVersion')
+
+    @property
+    def api(self):
```

Comment:
I think I'm :+1: on having a mixed-bag of coroutine and normal properties.  Forcing simple APIs like this that don't require I/O to be coroutines makes them more complex than necessary **and** adds un-necessary performance overhead.  :+1: